docs/pages/router/advanced/protected.mdx

```diff
@@ -143,7 +143,7 @@ export function AppLayout() {
 }
 ```
 
-Here, because the **index** screen is protected and the **protected** is **false**, the router redirects to the first available screen &mdash; **login**.
+Here, because the **index** screen is protected and the **guard** is **false**, the router redirects to the first available screen &mdash; **login**.
```

Comment:
```suggestion
In the above example, since the **index** screen is protected and the `guard` is **false**, the router redirects to the first available screen &mdash; **login**.
```

I'd like to update this sentence for clarity.